src/main/scala/com/fulcrumgenomics/sv/tools/SvPileup.scala

```diff
@@ -127,19 +151,31 @@ class SvPileup
           slop                     = slop
         )
 
+        val filteredEvidences = evidences.filter { ev =>
+          targets.forall { detector =>
```

Comment:
I think I would put the `targets.forall` on the outside (and make it `targets.foreach`) so that you don't even do the `filter()` call if you don't have targets to filter to.